src/mqueryfront/src/QueryStatus.js

```diff
@@ -233,9 +233,9 @@ class QueryStatus extends Component {
             </button>
         );
 
-        if (!this.state.job.total_files) {
-            progress = 0;
-            processed = "-";
+        if (isNaN(progress)) {
+            progress = 100
```

Comment:
If `this.state.job.total_files == 0` and the `this.state.job.status != "done"` then I think the progres should be 0 instead of 100 (the job was not actually completed succesfully).

Reply:
Technically the job was completed but nothing was found, so the progress is 100, the job is over.

Reply:
what @icedevml means is that total_files may be 0 and increase later, when the job is just starting, right?

Reply:
@msm-code yes, `total_files` is known from certain point, I've referenced appropriate code previously